CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
+
+* Enhancement: **New optional plugin parameter `is_callable`**.
+
+  Some plugins may have a free and a Premium version using different slugs. Using the `is_callable` plugin parameter allows for the Premium version to be recognized, even though the slug is set to the free version slug. Have a look at the `example.php` file for a working example.
+
+  For more information on what is considered a `callable`, please refer to the [Codex on callbacks].
+  
+  [#205] Props [Zack Katz].
+
+* **Admin Page improvements**:
+  - Plugins downloaded from an arbitrary external source are now labelled _"External Source"_. Previously they were labelled _"Private Repository"_ which could be confusing as the download URL did not have to point to a repository, let alone be private. [#372]
+  - Leverage the css styling of the Core standard `WP_List_Table` [#227]. Props [Shiva Poudel].
+  - Allow for moving the Admin Page to a different place in the menu. This is mainly to accommodate plugins using TGMPA as it is terribly illogical for the TGMPA page to be under the _"Appearance"_ menu in that case. This has been now been implemented in a way that Theme Check will not choke on it. [#310]
+
+* **Admin notices improvements**:
+  - For installs with both plugin(s) as well as theme(s) using TGMPA, notices will now be dismissable for each separately. This prevents a situation where a theme would have TGMPA included, the user has dismissed the notice about it, a plugin with TGMPA gets installed and the notice about it requiring certain other plugins is never shown. [#174] Thanks [Chris Howard] for reporting.
+  - Fixed: The reset of dismissed notices on `switch_theme` was only applied for the current user, not for all users. [#246]
+  - Fixed: Admin notices would show twice under certain circumstances. [#249], [#237] Thanks [manake] for reporting.
+
+* **Bulk Installer**:
+  - Fixed: If a bulk install was initiated using the bottom _"Bulk Actions"_ dropdown, the install page would display an outdated TGMPA plugin table at the bottom of the page after the bulk installation was finished. [#319]
+  - Fixed: The _"Show Details"_ links no longer worked. This was a regression briefly introduced in the `develop` branch. [#326]
+
+* **Theme Check compatibility**:
+  - Prevent _"The theme appears to use include or require"_ warning. [#262], [#258] Thanks [Tim Nicholson] for reporting.
+  - Preempt the disallowing of the use of the `add_theme_page()` function. See [the theme review meeting notes](https://make.wordpress.org/themes/2015/04/21/this-weeks-meeting-important-information-regarding-theme-options/) for further information on this decision. [#315]
+
+* **Miscellaneous fixes**:
+  - Leaner loading: TGMPA actions will now only be hooked in and run on the back-end (`is_admin() === true`).  [#357] Also most TGMPA actions will now only be hooked in if there's actually something to do for TGMPA. [#381]
```

Comment:
"...(`is_admin()` returns `true`)."

Let's avoid any chance to mention Yoda conditions ;-)